### DISCUSSION TITLE
Değeri ne kadar olucak

### DISCUSSION THREAD

**Discussion by Orfan74** [IBAX-io/go-ibax#1183] [Ideas]
Acaba topladığımız bu ibaxlar ne kadar edicek.

**Comment by Skips1** [comment]
Daha ibax toplayamadık malesef puan topluyoruz işte puana göre gelicek ki ben fazla bişey beklemiyorum 5 milyon dağıtılacak arzına göre çok ama çok az ve bunun yarısını en çok ref yapan kişilere vericekler yani çok bişey bekleme kanka
